### ISSUE TITLE
Proposal for new interface - tabbed gobans

### ISSUE THREAD

**Issue by pnprog** [pnprog/goreviewpartner#72]
Dear @jiangwuyou , @wonsiks , @fishcu and @BorisNA 

I am considering making a big interface change for GoReviewPartner. I value your opinions on how to use GoReviewPartner, so I would like to have your opinions on this modifications before implementing it.

I put a description with screenshots at this URL, please have a look: https://lifein19x19.com/viewtopic.php?p=233631#p233631

Then, if you have comments or ideas, you can directly reply here on github

Best regards

**Comment by jiangwuyou** [comment]
>So basically, I would like to use a "tab" system, à la Firefox:
Yeah.  But my suggestion is for the top DOS Windows. Because many Go players don't know too much about multi DOS Windows or English prompt message, I wish that GoReviewPartner can union multi DOS Windows into a "tab" panel. Your Settings panel is a good sample.

Another suggestion:
Because many Children are studing Go but doesn't know too much about English. I wish to split the analysis result so that we can translate it for Children.
For instance,
Move 1
Black to play, in the game, black played Q16
For this position, Leela Zero would play: D16

Black Value Network win probability:
 • before q16: 49.07%
 • after q16: 49.24% (+0.17pp)

**Comment by pnprog** [comment]
Hi @jiangwuyou ,
Thanks for your reply.

> Yeah. But my suggestion is for the top DOS Windows. Because many Go players don't know too much about multi DOS Windows or English prompt message, I wish that GoReviewPartner can union multi DOS Windows into a "tab" panel. Your Settings panel is a good sample.

I understand. This DOS windows is mainly here to help me debug the application, and understand errors when they appear on other users computer. Once the version 1 is finally ready, I will hide this windows, and maybe redirect everything into a log file or something. I use py2exe to package the application for Windows, it has the option to hide that DOS windows.

> Because many Children are studing Go but doesn't know too much about English. I wish to split the analysis result so that we can translate it for Children.
> For instance,
> Move 1
> Black to play, in the game, black played Q16
> For this position, Leela Zero would play: D16
> 
> Black Value Network win probability:
> • before q16: 49.07%
> • after q16: 49.24% (+0.17pp)

The translation is possible, in fact, most of it was already translated by you :)
![image](https://user-images.githubusercontent.com/25114663/42446989-d98c3fb0-83aa-11e8-8abf-f2e2b5cc61f3.png)

The missing string is number 10.3 in zh.po, ylu can update it anytime by clicking there: https://github.com/pnprog/goreviewpartner/edit/master/translations/zh.po

**Comment by wonsiks** [comment]
I think your tab idea is great and I also have another idea.
Merge of multi rsgf files which are analyzed by different bots for one game.
So people can see multi-bots' opinion when review the merged analysis.
Each bot's analysis can be showed in different tabs or can be merged in one goban.
I'm not sure about the way to show them altogether but just idea.
Let's think about it~
Cheers~
